tests/test_slogpy_console.py

```diff
@@ -0,0 +1,16 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog  # noqa: N813
+
+"""This is a setup function"""
+def setupfunction():
+    """Not to create a file for output"""
```

Comment:
nitpick: this is a very awkward syntax, also this is the place to say why you made this choice (the choice you made is good)